src/main/java/cofh/thermaldynamics/duct/attachments/servo/ServoFluid.java

```diff
@@ -102,9 +108,9 @@ public FilterLogic createFilterLogic() {
 
 	public IFluidHandler getTheTile() {
 
-		if (tile == null) {
+		if (theTile == null) {
```

Comment:
context: use `theTile` (the adjacent tile that should have a fluid capability) instead of `tile` (the fluiduct tile)